lms/djangoapps/mobile_api/video_outlines/serializers.py

```diff
@@ -218,5 +218,6 @@ def video_summary(course, course_id, video_descriptor, request, local_cache):
         "transcripts": transcripts,
         "language": video_descriptor.get_default_transcript_language(),
         "category": video_descriptor.category,
+        "only_on_web" : video_descriptor.unavailable_on_mobile,
```

Comment:
In fact, I would advocate having the video_module's field name be consistent with the corresponding course_module's field name: `mobile_available`.  Also, in the future, if we ever want the field to be inheritable, it will be easier.